src/components/Comments.jsx

```diff
@@ -1,12 +1,14 @@
 import { getComments } from '../../utils/api'
 import { useState, useEffect } from "react"
-import { formatDate} from "../../utils/utils.js"
+import { formatDate } from "../../utils/utils.js"
 import Votes from './Votes.jsx'
+import AddComment from './AddComment.jsx'
 
 function Comments({ article_id }) {
 
     const [comments, setComments] = useState([])
     const [loading, setLoading] = useState(true)
+    const [triggerReloadComments, setTriggerReloadComments] = useState(0)
```

Comment:
Does this need a count?

Reply:
I get why its here to trigger a reload, but what is the benefit of using numbers?